### ISSUE TITLE
Error on Spyder startup

### ISSUE THREAD

**Issue by canoetraveler** [spyder-ide/spyder#12645]
### What steps will reproduce the problem?

I get this error after upgrading to spyder 4.1.2.
It says there are missing dependencies, but when I go to install the missing modules, it says they are already installed. 

### Traceback
```python-traceback
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\app\mainwindow.py", line 989, in setup
    plugin = mod.PLUGIN_CLASS(self)
TypeError: 'bool' object is not callable
Traceback (most recent call last):
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\plugin.py", line 1083, in set_spyder_breakpoints
    cl.shellwidget.set_spyder_breakpoints()
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\widgets\debugging.py", line 143, in set_spyder_breakpoints
    CONF.get('run', 'breakpoints', {}))
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder_kernels\comms\commbase.py", line 538, in __call__
    call_dict, call_data, self._comm_id)
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\comms\kernelcomm.py", line 130, in _get_call_return_value
    with self.comm_channel_manager(comm_id, queue_message=not interrupt):
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\comms\kernelcomm.py", line 87, in comm_channel_manager
    timeout)
  File "C:\Users\mhagen\AppData\Local\Continuum\anaconda3\envs\spyder4\lib\site-packages\spyder\plugins\ipythonconsole\comms\kernelcomm.py", line 168, in _wait
    raise TimeoutError(timeout_msg)
TimeoutError: Timeout while waiting for comm port.
```

## Versions

* Spyder version: 4.1.2 
* Python version: 3.7.7
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 7

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.2.2 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.8.0 (OK)
jedi =0.15.2                   :  0.15.1 (NOK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.1 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.1 (NOK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.4.2 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.3 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.0.3 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.13 (OK)
matplotlib >=2.0.0             :  None (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  0.25.1 (OK)
scipy >=0.17.0                 :  1.3.1 (OK)
sympy >=0.7.3                  :  1.4 (OK)
```

**Comment by andfoy** [comment]
Hi @canoetraveler, thanks for reporting this issue, however, this is a duplicate of https://github.com/spyder-ide/spyder/issues/11889, which is going to be fixed on Spyder 4.1.4, to come out on the first days of June.

**Comment by canoetraveler** [comment]
Great to hear. I've gone back to Spyder 4.0.1 for now. So I'm good.

Mark Hagen (from mobile)
